storage/engine/db.cc

```diff
@@ -33,6 +33,10 @@
 #include "encoding.h"
 
 extern "C" {
+#include "_cgo_export.h"
+}  // extern "C"
```

Comment:
No, there is no significance to using a separate `extern "C"` block. I could have placed the `#include` in the one below.